Source/Coinbase/CoinbaseApiAuthenticator.cs

```diff
@@ -57,11 +58,15 @@ public void Authenticate(IRestClient client, IRestRequest request)
             var method = request.Method.ToString().ToUpper(CultureInfo.InvariantCulture);
 
             var body = string.Empty;
-
-            var param = request.Parameters.FirstOrDefault(p => p.Type == ParameterType.RequestBody);
-            if (param != null && param?.Value?.ToString() != "null" && !string.IsNullOrWhiteSpace(param?.Value?.ToString()))
-                body = param.Value.ToString();
-
+			if (request.Method != Method.GET) {
```

Comment:
@mitchcapper , are you doubley sure we need this check against `Method.GET`? IIRC, `param` variable would be `null` if the HTTP method was already `Method.GET`.